gui/build.py

```diff
@@ -79,6 +80,20 @@ def do_bundle():
     for dirname in ('build', 'dist'):
         if os.path.isdir(dirname):
             shutil.rmtree(dirname)
+    # create a intaller
+    env = dict(os.environ)
+    p = re.search('D:\\\\Python27\\\\Lib', env['PYTHONPATH'])
```

Comment:
Hm, not sure to understand what you are trying to do here...
I think that the python path "D:\\\\Python27" must be the configurable part. For this, you can do something like
python_lib_path = os.path.join(
    os.getenv('MOZREGUI_PYTHONPATH', "C:\\\\Python27"),
    "Lib"
)
Thiw way one can call this on the windows shell:
set MOZREGUI_PYTHONPATH=D:\Python27
python build.by bundle

If you prefer, instead of an environment variable you can use an option to the command line.